javaslang/generator/Generator.scala

```diff
@@ -1098,6 +1098,17 @@ def generateMainClasses(): Unit = {
               public T$j _$j() {
                   return _$j;
               }
+
+              /$javadoc
+               * Wither of the ${j.ordinal} element of this tuple.
+               *
+               * Note: The wither is NOT named {@code _$j} due to possible ambiguity when using method reference ($className::_$j).
+               *
+               * @return a copy of this tuple with a new value for the ${j.ordinal} element of this Tuple.
+               */
+              public $className$generics w$j(T$j t$j) {
```

Comment:
**Update:** please rename `w1`, ... to `update1`, ... This will align the names to the existing `Seq.update(int index, T element)` function